pkg/ui/src/components/NoteButton/NoteButton.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { ComponentMeta } from "@storybook/react";
+
+import { StorybookGrid, StorybookItem } from "../../utils/storybook";
+
+import NoteButton, { NoteButtonType, NoteButtonLabel } from "./NoteButton";
+
+export default {
+  component: NoteButton,
+  title: "NoteButton",
+} as ComponentMeta<typeof NoteButton>;
+
+export const Default = (): JSX.Element => (
+  <>
+    <h1 className="text-lg font-bold mb-4">Note In:</h1>
+    <StorybookGrid className="mb-16">
+      <StorybookItem>
+        <NoteButton type={NoteButtonType.In}></NoteButton>
+      </StorybookItem>
+    </StorybookGrid>
```

Comment:
No need to use the grid if there's only one element in the row (or grid for that matter)

Reply:
This can just be `<NoteButton {...}>`